builtin/providers/bitbucket/client.go

```diff
@@ -0,0 +1,65 @@
+package bitbucket
```

Comment:
Ok. Would be cool if we would have a good library. I know some provider authors have created very thin API libraries that were meant to be used by their providers. Anyway, thanks for the info 😄